content/en/community/hackathons/usoc22/behind-scenes/index.md

```diff
@@ -100,4 +100,5 @@ weigth: 20
 
 ### 12. Give Us Feedback
 
-{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Work-Items_12.-Give-Us-Feedback.md" markdown="true" >}}
+We want to know how to make the next sessions better.
+Fo this we need your [feedback](https://docs.google.com/forms/d/e/1FAIpQLSdpyEpCW0yxJBSDa7Iq95JUCe5YNnRpxOD31ol4ezpKWnqrhA/viewform).
```

Comment:
```suggestion
For this we need your [feedback](https://docs.google.com/forms/d/e/1FAIpQLSdpyEpCW0yxJBSDa7Iq95JUCe5YNnRpxOD31ol4ezpKWnqrhA/viewform).
```
My bad, I missed this in the patch file.

Reply:
fixed